app/views/step_by_step_pages/show/_actions.erb

```diff
@@ -32,10 +30,13 @@
         step_by_step_page_schedule_path(@step_by_step_page),
         class: %w(govuk-link govuk-link--no-visited-state app-link--right)
       %>
-    <% elsif !@step_by_step_page.steps_have_content? %>
-      <p class="govuk-body">Step by steps cannot be published until all steps have content.</p>
-      <%= preview_link %>
     <% elsif @step_by_step_page.has_draft? %>
+      <% if !@step_by_step_page.status.submitted_for_2i? && (current_user.has_permission? "Unreleased feature") %>
```

Comment:
Maybe it would be better to have this wrapped in a method and add unit tests as Leena suggested here: https://github.com/alphagov/collections-publisher/pull/678#discussion_r312082528

Reply:
Sounds like a good suggestion, but the `current_user` permissions will make these helper methods messy and will introduce more complexity in both code and tests. I tried to extract them on separate lines in the view files so we can remove them with the feature flag.